vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -256,15 +256,22 @@ private static <T> String modelToPresentation(ComboBox<T> comboBox,
     /**
      * Whether the component should block user input that does not match the
      * configured pattern
+     *
+     * @deprecated Since 23.2, this API is deprecated.
      */
+    @Deprecated
     public boolean isPreventInvalidInput() {
         return getElement().getProperty("preventInvalidInput", false);
     }
 
     /**
      * Sets whether the component should block user input that does not match
      * the configured pattern
+     *
+     * @deprecated Since 23.2, this API is deprecated in favor of
+     *             {@code setAllowedCharPattern(charPattern)}
```

Comment:
```suggestion
     *             {@link #setAllowedCharPattern(String)}
```
We can directly link to the method.

Reply:
Thanks, done.